wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
-# Beat snap divisor
+# Beat Snap Divisor
 
-The Beat Snap Divisor is used to limit where notes can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the notes, that is, more notes can be placed in the same measure.
+![Beat Snap Divisor in the editor](img/beat_snap_divisor.png)
 
-Each of the settings put different colours of ticks on the [Timelines](/wiki/Beatmap_Editor/Timelines). The ticks are as follows (for [Standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch)):
+The **Beat Snap Divisor** is a built-in editor functionality that ensures every [hit objects](/wiki/Hit_Object) are placed proportionally to the song's beats. It can be found in the upper-right portion of the editor screen, where it corresponds to the granularity of the objects on the [editor timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes can be placed in the same measure and vice versa. 
 
-## osu!, osu!taiko and osu!catch
+The osu! editor supports eleven different Beat Snap Divisor settings as follows:
 
-### Divisor 1/1
+## 1/1 snap divisor
 
-![Divisor 1/1](/wiki/shared/BSD_1_1b.jpg "Divisor 1/1")
+![1/1 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_1b.jpg "1/1 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-- Full Beats
-- White ticks only
-- Best used for Easy difficulty
+![1/1 snap divisor in the osu!mania editor](/wiki/shared/1_1_m.jpg "1/1 snap divisor in the osu!mania editor")
 
-### Divisor 1/2
+The 1/1 snap divisor allows for hit objects to be placed on every *full beats*. It is being represented by white-coloured ticks on the editor timeline.
 
-![Divisor 1/2](/wiki/shared/BSD_1_2.jpg "Divisor 1/2")
+Due to its simplicity and ease of use, this snap divisor is most commonly used while mapping Easy difficulties.
 
-- Half Beats
-- Adds red ticks
+## 1/2 snap divisor
 
-### Divisor 1/3
+![1/2 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_2.jpg "1/2 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-![Divisor 1/3](/wiki/shared/BSD_1_3.jpg "Divisor 1/3")
+![1/2 snap divisor in the osu!mania editor](/wiki/shared/1_2_m.jpg "1/2 snap divisor in the osu!mania editor")
 
-- Triplets*
-- Has two purple ticks between each white tick
-- Compound timing
+The 1/2 snap divisor allows for hit objects to be placed on every *half beats* and its multiples (e.g. 1/2). It is being represented by red-coloured ticks on the editor timeline.
```

Comment:
```suggestion
The 1/2 snap divisor allows for hit objects to be placed on *half beats* and its multiples (e.g. 1/2). It is being represented by red-coloured ticks on the editor timeline.
```
is it not already obvious that 1/2 is a "multiple" of 1/2?

Reply:
the word "multiple" here is also intentional because by setting the Beat Snap Divisor you're not *exclusively locked* to red tick snaps; you can also put objects in a 2/2 snap (or 1/1, as we know it) which is obv a multiple of 1/2